homeassistant/components/forked_daapd/media_player.py

```diff
@@ -715,22 +716,23 @@ async def _async_announce(self, media_id: str):
             )
             if saved_state == MediaPlayerState.PLAYING:
                 await self.async_media_play()
-        else:  # restore stashed queue
-            if saved_queue:
-                uris = ""
-                for item in saved_queue["items"]:
-                    uris += item["uri"] + ","
-                await self._api.add_to_queue(
-                    uris=uris,
-                    playback="start",
-                    playback_from_position=saved_queue_position,
-                    clear=True,
-                )
-                await self._api.seek(position_ms=saved_song_position)
-                if saved_state == MediaPlayerState.PAUSED:
-                    await self.async_media_pause()
-                elif saved_state != MediaPlayerState.PLAYING:
-                    await self.async_media_stop()
+            return
+        if saved_queue:  # restore stashed queue
```

Comment:
I think you could outdent here also.
```suggestion
        if not saved_queue:
            return
        # restore stashed queue
```